index.html

```diff
@@ -4,36 +4,39 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.0.0/css/all.css" integrity="sha384-3B6NwesSXE7YJlcLI9RpRqGf2p/EgVH8BgoKTaUrmKNDkHPStTQ3EyoYjCGXaOTS" crossorigin="anonymous" />
     <title>Portfolio Project</title>
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
    <header class="logo-div">
-        <div>
             <div class="logo"><a href="#first-section">Eligrand</a></div>
-        </div>
         <nav>
           <img src="images/Union.png" alt="image" class="nav-image">
         <div class="navbar-items">
-            <ul>
-                <li><a href="#" class="portfolio">Portfolio</a></li>
-                <li><a href="#" class="about">About</a></li>
-                <li><a href="#" class="contact">Contact</a></li>
+            <ul class="nav-container">
+                <li><a href="#" class="list-item">Portfolio</a></li>
+                <li><a href="#" class="list-item">About</a></li>
+                <li><a href="#" class="list-item">Contact</a></li>
                 </ul>
         </div>
       </nav>
 </header> 
 <main>
     <section class="wrapper" id="first-section">
-        <article class="intro">
-            <h1 class="h1-intro">I'm Eligrand<br/>
+        <article class="intro intro-wrapper">
+            <h1 class="h1-intro mobile">I'm Eligrand<br/>
               Glad to see<br />
-               you!
+              you!
+            </h1>
+            <h1 class="h1-intro desktop">I'm Eligrand<br/>
+              Glad to see
+              you!
             </h1>
                 <p class="p-intro">
-                    I’m a software developer! I can help you build a product , 
-                    feature or website Look through 
-                    some of my work and experience! If you like what you see and have a project you need
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through<br />
+                    some of my work and experience! If you like what you see and have a project you need<br />
                      coded, don’t hesitate to contact me.
                 </p>
```

Comment:
- [ ] Kindly make sure you stick to the design as much as you can, You may consider using the same typo(font) for your text
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201397798-c6bca345-d814-496d-bfa9-12db1cd6ac8c.png)


Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201397722-9e02ce95-65cf-470c-b61c-667da7239743.png)